index.js

```diff
@@ -87,6 +87,12 @@ function create (delay, fn) {
           el.dispatchEvent(new window.KeyboardEvent(event || 'keydown', { key: c }))
         }
         return t.delay()
+      },
+      async appendChild (el) {
+        if (!el) return
+        t.element.appendChild(el)
+        await t.onload(el)
+        return t.delay()
```

Comment:
```suggestion
```

Reply:
Lets not have a delay hook here.

Reply:
Thanks :) I didn't see that onload hook had his own delay hook.